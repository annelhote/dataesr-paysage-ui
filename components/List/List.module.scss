.List {
    max-height: 320px;
    overflow-x: scroll;
    width: 100%;
}
