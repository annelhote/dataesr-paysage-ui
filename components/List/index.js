import style from "./List.module.scss";

export default function List({ children }) {
  return (
    <section className={style.List}>
      <ul className="p-0">{children}</ul>
    </section>
  );
}
