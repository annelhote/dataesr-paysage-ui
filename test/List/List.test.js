import React from "react";
import renderer from "react-test-renderer";

import List from "../../components/List";

it("renders correctly List", () => {
  const tree = renderer
    .create(
      <List>
        <p>child</p>
      </List>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
