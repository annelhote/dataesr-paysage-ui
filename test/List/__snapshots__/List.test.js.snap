// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly List 1`] = `
<section
  className="List"
>
  <ul
    className="p-0"
  >
    <p>
      child
    </p>
  </ul>
</section>
`;
